odoo/tools/misc.py

```diff
@@ -1361,6 +1362,18 @@ def format_duration(value):
     return '%02d:%02d' % (hours, minutes)
 
 
+def _crypt(msg, crypt_ctx=None):
```

Comment:
why didn't you used `_crypt_context` and `encrypt` btw?

Reply:
it is exactly what this function is doing @mart-e 
Is is an helper ;)

Reply:
that was a bit my point, is an helper worth it when it takes two lines otherwise? let's say we want to deprecate `pbkdf2_sha512` at one point, this is one more place to change it

Reply:
> let's say we want to deprecate pbkdf2_sha512 at one point, this is one more place to change it


Or it could be the only place to replace if we created earlier.